libs/domain/picking/offline/sw-feature.ts

```diff
@@ -0,0 +1,11 @@
+import { AppFeature, AppInitializer } from '@spryker-oryx/core';
+import { SwPushInitializerService } from './services';
+
+export class SwOfflinePickingFeature implements AppFeature {
```

Comment:
This feature should extend `OfflinePickingFeature` and override plugins prop to remove them.
Then in the main SW feature you should only use `SwOfflinePickingFeature` without a hacking of `OfflinePickingFeature` currently done there.